src/sound/audio_stream/msg/AudioData.msg

```diff
@@ -0,0 +1 @@
+uint8[] data
```

Comment:
Changed in another PR #193 . This is the raw audio stream data. It's currently used by speech recognition.